cli/.js/src/main/scala/aqua/js/UtilsTypes.scala

```diff
@@ -0,0 +1,24 @@
+package aqua.js
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{JSExportAll, JSImport}
+
+object Meta {
+
+  // get `import`.meta.url info from javascript
```

Comment:
Why do you want to get it?

Reply:
Please add comments (what does createRequire mean is also unclear) & rename `UtilsTypes` file to smth more meaningful